docs/snippets/common/storybook-auto-docs-baseline-example.custom-title.mdx.mdx

```diff
@@ -0,0 +1,30 @@
+
+```mdx
+{/* Button.mdx */}
+
+import { Meta } from '@storybook/blocks';
+
+<Meta title="Button" />
+
+# Definition
+
+Button is a clickable interactive element that triggers a response.
+
+You can place text and icons inside of a button.
+
+Buttons are often used for form submissions and to toggle elements into view.
+
+## Usage
+
+The component comes in different variants such as `primary`, `secondary`, `large` and `small` which you can use to alter the look and feel of the button.
+
+## Inputs
+
+Button has the following properties:
+
+- `primary` - If `true`, the button will have a primary style.
```

Comment:
This feels like an anti-pattern, because the ArgTypes or Controls blocks are a _much_ better means of documenting these.